llm_bench/load_test.py

```diff
@@ -688,12 +710,30 @@ def _maybe_randomize(prompt):
             )
 
         if isinstance(self.input, str):
-            return _maybe_randomize(self.input), None
+            prompt = self.input
+            images = None
         else:
             item = self.input[random.randint(0, len(self.input) - 1)]
             assert "prompt" in item
-            return _maybe_randomize(item["prompt"]), item.get("images", None)
+            prompt = item["prompt"]
+            images = item.get("images", None)
+        
+        prompt = _maybe_randomize(prompt)
+
+        if self.prompt_images:
+            images = self.prompt_images
+            prompt = self.insert_image_placeholders(prompt, len(images))
+
+        return prompt, images
 
+    def insert_image_placeholders(self, prompt, num_images):
+        """
+        Insert <image> placeholders evenly throughout the prompt.
+        E.g., for a prompt of Hello World, and 3 images, the output should be <image>Hello <image>World<image>
+
+        Images are spaced out evenly based on on character length
```

Comment:
Just going to do character length for now since it's the easiest. I believe in the worst case, we only affect token counts by num_images (where num_image placeholders are placed in the middle of a token, splitting the token into 2).